pkg/controllers/deprovisioning/controller.go

```diff
@@ -115,32 +116,13 @@ func (c *Controller) Reconcile(ctx context.Context, _ reconcile.Request) (reconc
 	// Attempt different deprovisioning methods. We'll only let one method perform an action
 	isConsolidated := c.cluster.Consolidated()
 	for _, d := range c.deprovisioners {
-		candidates, err := GetCandidates(ctx, c.cluster, c.kubeClient, c.clock, c.cloudProvider, d.ShouldDeprovision)
+		success, err := c.deprovision(ctx, d)
 		if err != nil {
-			return reconcile.Result{}, fmt.Errorf("determining candidates, %w", err)
+			logging.FromContext(ctx).Infof("could not deprovision with %q, %s", d, err)
```

Comment:
Don't we want to wrap here? 
```suggestion
			logging.FromContext(ctx).Infof("could not deprovision with %q, %w", d, err)
```

Reply:
Can't wrap in a logging statement.